source/Target/SwiftLanguageRuntime.cpp

```diff
@@ -474,10 +481,10 @@ static bool GetObjectDescription_ObjectCopy(Process *process, Stream &str,
       process->GetThreadList().GetSelectedThread()->GetSelectedFrame();
   auto *swift_ast_ctx =
       llvm::dyn_cast_or_null<SwiftASTContext>(static_type.GetTypeSystem());
-  if (swift_ast_ctx)
-    static_type =
-        swift_ast_ctx->MapIntoContext(frame_sp,
-                                      static_type.GetOpaqueQualType());
+  if (swift_ast_ctx) {
+    SwiftASTContextLock lock(GetSwiftExeCtx(object));
+    static_type = runtime->DoArchetypeBindingForType(*frame_sp, static_type);
```

Comment:
This is good for basically the same reason as above. We're asking the object to print its description using the expression evaluator, so somebody has to bind the archetypes and it might as well be us.